docs/userguide/torch_forecasting_models.md

```diff
@@ -213,48 +264,191 @@ model.fit(...)
 best_model = model.load_from_checkpoint(model_name='MyModel', best=True)
 ```
 
-You can also save or load manually:
+The automatic checkpoints can also be saved directly on remote file systems, including cloud storage providers such as S3 on AWS, GCA on Google Cloud and ADL on Azure (see [detailed documentation](https://lightning.ai/docs/pytorch/latest/common/checkpointing_advanced.html)):
```

Comment:
I did not take the time to setup a remote file system to test this feature but since this depend on the trainer alone, it might actually work out of the box 🤞

Reply:
Tested whether this works for a local path, and it does not.
While PyTorch Lightning checkpointing might work, the `ckpt_path` has to be passed to trainer.fit(), which we currently only do when loading a model from checkpoint with `TFM.load_from_checkpoint()`. This assumes that there is the base save file for the TorchForecastingModel, which would not be saved to the cloud.

I'll remove it for now.